docs/en/sql-reference/aggregate-functions/index.md

```diff
@@ -18,6 +18,10 @@ ClickHouse also supports:
 
 During aggregation, all `NULL`s are skipped. If the aggregation has several parameters it will ignore any row in which one or more of the parameters are NULL.
 
+There are a few exceptions to this rule:
+  - Both [`first_value`](../../sql-reference/aggregate-functions/reference/first_value.md) and [`last_value`](../../sql-reference/aggregate-functions/reference/last_value.md) support modifiers that respect NULLs (`first_value(b) ignore nulls`).
```

Comment:
We also need to add aliases in ClickHouse style: `firstValue`, `lastValue`.

Reply:
We already have `any`, `any_value`, `first_value` all pointing to the same aggregate function and `ANY_VALUE` I guess it's there for compatibility with Postgres, Mysql, etc. In fact any of them supports `IGNORE/RESPECT NULLS`.